core/src/edu/asu/c3simulator/widgets/HomeButton.java

```diff
@@ -0,0 +1,45 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.screens.AllManagementScreens;
+/**
+ * Creates a Home button, places it in the bottom left of the screen. Changes the screen to MainHub when clicked.
+ * 
+* @author Reigel, Justin 
+* 
+*/
+public class HomeButton extends Table{
+	
+	private Image homeIcon;
+	private float homeLeft;
+	private float homeBottom;
+	
+	public HomeButton(final Game game)
+	{
+		super();
+		
+		FileHandle iconLocation = Gdx.files.internal("images/home-button.png");
+		Texture iconTexture = new Texture(iconLocation);
+		this.homeIcon = new Image(iconTexture);
+		add(homeIcon).size(150);
+		homeLeft = this.getPrefWidth()/2;
+		homeBottom = this.getPrefHeight()/2;
+		this.setPosition(homeLeft, homeBottom);
+		this.addListener(new ClickListener(){
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				game.setScreen(AllManagementScreens.MAIN_HUB.getInstance());
+			}
+		});
+	}
```

Comment:
Classes add overhead, and are bulky. Generally, if a class is going to be instantiated, we should get some value from the expense of having the class (i.e. it should provide some functionality, or data relationship that isn't achieved otherwise). 

In this case, the class doesn't actually do anything, except assemble a table.